text/2018-12-19-unified-log-format.md

```diff
@@ -0,0 +1,207 @@
+# Unified Log Format
+
+## Summary
+
+This RFC stipulates a unified log format called "TiDB Log Format" to be used
+across all TiDB software, i.e. TiDB, TiKV, and PD.
+
+## Motivation
+
+A unified log format makes it easy to collect (i.e. Fluentd) logs into a central
+storage (i.e. ElasticSearch) and allows users to query logs in a structural way
+later.
+
+## Detailed Design
+
+A TiDB Log Format file contains a sequence of *lines* containing UTF-8
+characters terminated by either the sequence LF or CRLF. Each line contains a
+*Log Header Section* and a *Log Fields Section*, concatenated by one whitespace
+character U+0020 (SPACE).
+
+### Log Header Section
+
+The Log Header Section is in the following format:
+
+```text
+[date_time] [LEVEL] [source_file:line_number]
+```
+
+- `date_time`: The human readable time that the log line is generated in the
+  local time zone in the format `YYYY/MM/dd HH:mm:ss.SSS Z`.
```

Comment:
This doc is using momentjs's token format (as explained in Line 32): http://momentjs.com/docs/#/displaying/format/

Looks like in thousands of years we don't need to worry about it?

Reply:
> This doc is using momentjs's token format

To be honest, it feels odd the RFC depends on a js library instead of another RFC/Standard.

> Looks like in thousands of years we don't need to worry about it?

 It happens every year.